server/src-lib/Hasura/RQL/Types/RemoteSchema.hs

```diff
@@ -30,6 +30,7 @@ data RemoteSchemaInfo
   { rsUrl              :: !N.URI
   , rsHeaders          :: ![HeaderConf]
   , rsFwdClientHeaders :: !Bool
+  , rsTimeoutSeconds   :: !Int
```

Comment:
Consider maybe using [`DiffTime`](https://hackage.haskell.org/package/time-1.9.3/docs/Data-Time-Clock.html#t:DiffTime) here to add a small amount of type safety? The main advantage, imo, is that the units are unambiguous (a `DiffTime` is always treated to mean seconds when converted to/from other numeric types).

Reply:
Turns out `DiffTime` is not an instance of `Lift` which is creeping into 20 odd places.

Reply:
Odd. I guess that’s alright, then, it’s not a big deal.